scripts/checks.sh

```diff
@@ -5,3 +5,20 @@ set -o pipefail
 npm run test:lint
 npm run prettier:ci
 npm run test
+
+pushd integration-test/nodejs
+sls deploy
+sls invoke -f hello
```

Comment:
would this catch errors from the invocation? i.e. if there's a problem with the wrapped function, and the invocation errors, would this error and fail the test as well?

Reply:
```
set -e
set -o pipefail
```

Does the trick, the script fails on any command with exit code > 0

Reply:
Should have framed my question better. Does `sls invoke -f` return a non-0 exit code when the invocation errors?

Reply:
Yes, checked it by throwing an exception from within the lambda and saw that the exit code > 0

Reply:
👍